src/main/java/org/myrobotlab/service/Runtime.java

```diff
@@ -1399,7 +1402,7 @@ public static final long getTotalMemory() {
    */
   public static final synchronized long getUniqueID() {
     ++uniqueID;
-    return uniqueID;
+    return System.currentTimeMillis();

```

Comment:
This seems odd..  current time milliseconds isn't very unique..  why not a uuid like the fixme mentions above?

Reply:
heh, sure ... I thought the same thing,
but considering it will be changing the core message from long to string I figured it be worth investigating in a separate pull request ;)